UIKitDZ/UIKitDZ/ChoseBookController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ViewController.swift
+//  UIKitDZ
+//
+//  Created by Valery on 22.06.2021.
+//
+
+import UIKit
+
+final class ChosenBookController: UIViewController {
+
+    //MARK: - Visual Components
+    private var logo = UIImageView(image: UIImage(named: "logobook"))
+    private var slogan = UILabel()
+    private var bookImage = UIImageView(image: UIImage(named: "platon"))
+    private var button = UIButton()
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        //MARK: - Initializers
```

Comment:
```suggestion
        //MARK: - Initializers
```
не пишем внутри методов а выше